chart/templates/_helpers.tpl

```diff
@@ -73,3 +73,14 @@ Usage:
         {{- tpl (.value | toYaml) .context }}
     {{- end }}
 {{- end -}}
+
```

Comment:
I would propose to rise a pr with this in our [common chart](https://github.com/keptn/lifecycle-toolkit-charts/tree/main/charts/common) and import it in this pr after release

Reply:
okay 👍